app/views/common/ExplanationScreen.tsx

```diff
@@ -0,0 +1,197 @@
+import { Typography } from '../../components/Typography';
+
+import {
+  Buttons,
+  Colors,
+  Layout,
+  Spacing,
+  Iconography,
+  Typography as TypographyStyles,
+} from '../../styles';
+
+export enum IconStyle {
+  Blue,
+  Gold,
+}
+
+type ExplanationScreenContent = {
+  icon: string;
+  header: string;
+  body: string;
+  primaryButtonLabel: string;
+  secondaryButtonLabel?: string;
+  backgroundImage: ImageSourcePropType;
+};
+
+type ExplanationScreenStyles = {
+  headerStyle?: TextStyle;
+  bodyStyle?: TextStyle;
+  primaryButtonContainerStyle?: ViewStyle;
+  primaryButtonTextStyle?: TextStyle;
+  secondaryButtonContainerStyle?: ViewStyle;
+  secondaryButtonTextStyle?: TextStyle;
+  backgroundStyle?: ViewStyle;
+  iconStyle: IconStyle;
+};
+
+type ExplanationScreenActions = {
+  primaryButtonOnPress: () => void;
+  secondaryButtonOnPress?: () => void;
+};
+
+interface ExplanationScreenProps {
+  explanationScreenContent: ExplanationScreenContent;
+  explanationScreenStyles: ExplanationScreenStyles;
+  explanationScreenActions: ExplanationScreenActions;
+}
+
+const ExplanationScreen = ({
+  explanationScreenContent,
+  explanationScreenStyles,
+  explanationScreenActions,
+}: ExplanationScreenProps): JSX.Element => {
+  useStatusBarEffect('dark-content');
+
+  function getIconStyle(iconStyle: IconStyle) {
+    switch (iconStyle) {
+      case IconStyle.Blue:
+        return styles.blueIcon;
+      case IconStyle.Gold:
+        return styles.goldIcon;
+    }
+  }
+
+  const primaryButtonTextStyles = {
+    ...styles.primaryButtonText,
+    ...explanationScreenStyles.primaryButtonTextStyle,
+  };
+
+  const secondaryButtonTextStyles = {
+    ...styles.secondaryButtonText,
+    ...explanationScreenStyles.secondaryButtonTextStyle,
+  };
+
+  const headerStyles = {
+    ...styles.headerText,
+    ...explanationScreenStyles.headerStyle,
+  };
+
+  const contentStyles = {
+    ...styles.contentText,
+    ...explanationScreenStyles.bodyStyle,
+  };
+
+  const smallScreenWidth = Layout.screenWidth <= Layout.smallScreenWidth;
+
+  return (
+    <View style={styles.outerContainer}>
+      {smallScreenWidth ? null : (
+        <ImageBackground
+          source={explanationScreenContent.backgroundImage}
+          style={[styles.background, explanationScreenStyles.backgroundStyle]}
+        />
+      )}
+      <View style={styles.content}>
+        <ScrollView
+          alwaysBounceVertical={false}
+          style={styles.innerContainer}
+          contentContainerStyle={{ paddingBottom: Spacing.large }}>
+          <View style={getIconStyle(explanationScreenStyles.iconStyle)}>
+            <SvgXml xml={explanationScreenContent.icon} />
+          </View>
+          <Typography style={headerStyles}>
+            {explanationScreenContent.header}
+          </Typography>
+          <Typography style={contentStyles}>
+            {explanationScreenContent.body}
+          </Typography>
+        </ScrollView>
+        <TouchableOpacity
```

Comment:
I don't think this should be a `TouchableOpacity`, but a `Button` component -- these buttons need to communicate more states on some screens, like loading, which they currently do, e.g. in our workflow on ExportTemplate (Which it looks like to me that this will encapsulate by generalizing the templates?)

```js
          <Button
            label={nextButtonLabel}
            onPress={onNext}
            loading={buttonLoading}
          />
```

Reply:
Hey @JacobJaffe, can you clarify why this should be a `Button`? I almost always use `TouchableOpacity` instead of `Button` because `Button` doesn't allow for custom styling. [See React Native docs.](https://reactnative.dev/docs/button)

From the linked article:
> If this button doesn't look right for your app, you can build your own button using TouchableOpacity or TouchableNativeFeedback.

Reply:
Re: `ExportTemplate`, how do you feel about getting this in place first and perhaps making this configurable for additional screens in the future if it makes sense?

Reply:
Ah, to clarify I mean our `components/button`

Reply:
> Re: ExportTemplate, how do you feel about getting this in place first and perhaps making this configurable for additional screens in the future if it makes sense?

Sure, I just want to ensure we're not introducing regressions in the future with these migrations from our button component

Reply:
Ah I see @JacobJaffe, the latest commit refactors to use the custom `Button` component.